tests/api/test_sources_endpoint.py

```diff
@@ -0,0 +1,141 @@
+import json
+import pytest
+
+from share.models import Source, ShareUser
+
+
+PROPER_ICON_URL = 'https://staging-cdn.osf.io/preprints-assets/bitss/square_color_no_transparent.png'
```

Comment:
We shouldn't be making HTTP requests in tests, can you mock the urls?
There are a couple libraries that are pretty nice, httpretty is my favorite. Sherif has been using mock-requests.